src/plone/recipe/zope2instance/recipe.py

```diff
@@ -598,9 +604,13 @@ def is_rs_option(name):
     def build_wsgi_ini(self):
         options = self.options
         wsgi_ini_path = os.path.join(options['location'], 'etc', 'wsgi.ini')
+        listen = options.get('http-address', '0.0.0.0:8080')
+        if ':' not in listen:
+            listen = '0.0.0.0:{}'.format(listen)
         options = {
             'location': options['location'],
-            'http_address': options.get('http-address', '8080'),
+            'http_address': listen,
+            'threads': options.get('threads', 4),
```

Comment:
Above, Zserver has 2 threads. Here WSGI has 4. Is it by intend?

Reply:
4 is the default for waitress, s. https://docs.pylonsproject.org/projects/waitress/en/latest/arguments.html. I didn't want to offer less than that.